README.md

```diff
@@ -6,10 +6,15 @@ Plugin for OBS Studio to move source to a new position during scene transition
 https://obsproject.com/forum/resources/move-transition.913/
 
 # Build
-- Build OBS Studio: https://obsproject.com/wiki/Install-Instructions
-- Check out this repository to plugins/move-transition
-- Add `add_subdirectory(move-transition)` to plugins/CMakeLists.txt
-- Rebuild OBS Studio
+1. In-tree build
+    - Build OBS Studio: https://obsproject.com/wiki/Install-Instructions
+    - Check out this repository to plugins/move-transition
+    - Add `add_subdirectory(move-transition)` to plugins/CMakeLists.txt
+    - Rebuild OBS Studio
+
+1. Stand-alone build (Linux only)
```

Comment:
I'm not actually sure if it's Linux specific, but I assume OBS doesn't provide headers and libraries for Windows and Mac versions